### DISCUSSION TITLE
Unable to expand entity classes in DB editor

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2937] [Support]
After I start the DB editor for the second time after starting the toolbox, I'm unable to view e.g. all "nodes" in the entity tree. The first time it works.

![image](https://github.com/user-attachments/assets/e22be3b9-5419-455d-af17-a2c004e07318)

**Comment by soininen** [comment]
Are you using the latest Toolbox, `spinedb_api` etc.? I vaguely remember fixing a similar issue some time ago.

**Reply by Mastomaki** [reply]
![image](https://github.com/user-attachments/assets/dac0059a-02b4-4b8d-985d-793e1fd3f296)

**Reply by soininen** [reply]
The package versions look OK. I tried with same Python version but did not see anything out of ordinary. I opened different data stores multiple times in different projects and everything seemed to work as expected. Are there some specific steps you have to take to make the issue appear? Or is it a specific database? If it is the latter, would it be possible to provide the database so I can check it out?

**Reply by Mastomaki** [reply]
Seems to be working now! 🚀
